src/shared/components/common/markdown-textarea.tsx

```diff
@@ -260,8 +260,12 @@ export class MarkdownTextArea extends Component<
               <LanguageSelect
                 iconVersion
                 allLanguages={this.props.allLanguages}
+                // Only set the selected language ID if it exists as an option
+                // in the dropdown; otherwise, set it to 0 (Undetermined)
                 selectedLanguageIds={
-                  languageId ? Array.of(languageId) : undefined
+                  languageId && this.props.siteLanguages.includes(languageId)
+                    ? Array.of(languageId)
```

Comment:
Is there any reason `Array.of` is better here than just throwing it in some square brackets, or it this just to keep parity with what was there before?

Reply:
Nope, just left it like that. I agree it feels better with brackets, which works fine, will commit.